docs/snippets/cpu/dynamic_shape.py

```diff
@@ -1,11 +1,13 @@
 # Copyright (C) 2022 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
+from snippets import get_model
 
-from openvino.runtime import Core
+model = get_model()
 
 #! [static_shape]
-core = Core()
-model = core.read_model("model.xml")
+import openvino as ov
```

Comment:
in some places `import` statement is out of code snippet. Do we have an alignment here?

@jiwaszki

Reply:
As I can see in some snippets we have this special marking `#! [import]`
But actually in generated docs is used only one -- [link](https://github.com/openvinotoolkit/openvino/blob/master/docs/OV_Runtime_UG/integrate_with_your_application.md).

my suggestion is to move `import openvino as ov` out of code snippets but keep there special imports